ember-cli/2018-02/february-08.md

```diff
@@ -0,0 +1,7 @@
+# ember-cli Meeting Notes - 2018-02-08
+
+## Attendees
+
+## Agenda
```

Comment:
@rwjblue @kellyselden @kratiahuja @kategengler

Reply:
discuss face to face agenda https://paper.dropbox.com/doc/Roadmap-to-discuss-xCNSfLwiElNtNGqUEBP5k